packages/nimble-components/src/chip/specs/README.md

```diff
@@ -0,0 +1,200 @@
+
+### Risks and Challenges
+
+None
+
+### Prior Art/Examples
+
+SLE keywords display:
+
+![](./spec-images/SLE-keywords.PNG)
+
+SLE axis selection:
+
+![](./spec-images/SLE-axis-selection.PNG)
+
+Nigel chat attachment chip:
+
+![](./spec-images/Nigel-attachment-chip.PNG)
+
+[Infragistics Chip component](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/inputs/chip)
+
+---
+
+## Design
+
+### API
+
+_The key elements of the component's public API surface:_
+
+- _Component Name_: `nimble-chip`
+- _Props/Attrs_:
+    - `prevent-remove` - set to hide the remove button
+    - `appearance` - supports `outline` and `block` appearances
+    - `disabled`
+- _Methods_
+- _Events_
+    - `remove` - fired when the chip remove button is pressed (or \<Del\> or \<Backspace\> is pressed), and `prevent-remove` hasn't been set.
+- _Slots_
+    - `start` - content placed to the left of the chip text
+    - (default) - for the primary label text
+- _CSS Classes and CSS Custom Properties that affect the component_
+    - not supported
+
+### Examples
+
+```html
+<nimble-chip> "Option Label" </nimble-chip>
+```
+
+![](./spec-images/Chip-normal.PNG)
+
+```html
+<nimble-chip prevent-remove> "Option Label" </nimble-chip>
+```
+
+![](./spec-images/Chip-no-remove.PNG)
+
+```html
+<nimble-chip>
+    <nimble-icon-check slot="start"> </nimble-icon-check>
+    "Option Label"
+</nimble-chip>
+```
+
+![](./spec-images/Chip-checked.PNG)
+
+### Anatomy
+
+```html
+<template aria-disabled="${x => x.disabled}">
+    <div class="control" part="control">
+        ${startSlotTemplate(context, definition)}
+        <span class="content" part="content">
+            <slot></slot>
+        </span>
+        ${endSlotTemplate(context, definition)}
+        ${when(x => !x.preventRemove, html<Chip>`
+            <nimble-button content-hidden @click="${x => x.handleRemoveClick()}">
+                <nimble-icon-times>
+                </nimble-icon-times>
+            </nimble-button>
+        `)}
+    </div>
+</template>
+```
+
+While the template will have a named `end` slot, this will not be documented, and only be populated with the error icon as needed, following existing Nimble patterns.
+
+### Angular integration
+
+An Angular wrapper will be created for `nimble-chip`. No `ControlValueAccessor` is needed.
+
+### Blazor integration
+
+Blazor integration for the `nimble-chip` will be provided.
+
+### Visual Appearance
+
+See Figma document link at the top of this document.
+
+---
+
+## Implementation
+
+### States
+
+The CSS `hover` and `focusVisible` states will have specific stylings (as seen in the UX designs), in addition to styling capturing the `disabled` attribute state.
+
+### Accessibility
+
+_Consider the accessibility of the component, including:_
+
+- _Keyboard Navigation and Focus_
+    - the chip component is focusable, as is the remove button (when present).
+- _Form Input_
+    - N/A
+- _Use with Assistive Technology_
```

Comment:
Do we need to consider the `role` of the chip? It would be good to make a statement about what it will be (even if we don't need to explicitly set it in the template).

Reply:
Added a statement asserting that we don't need to set the `role`. I'm not going to die on this hill though. It seems if we do set it it would be to `status` (until we make the chip component interactive).